docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,35 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server provides a means of securely decrypting payload data for display by Temporal user interfaces.
+
+On the Temporal Platform, you can use the WebUI or the [tctl plugin](/docs/tctl/index) to request data from the Temporal Cluster. This request is passed through a client's [Data Converter](/docs/concepts/what-is-a-data-converter), which sends and receives API calls from the Remote Codec Server. These calls are then passed from the client to the cluster, where a successful request passes an encoded payload back through the platform.
```

Comment:
A Data Converter resides in the Codec Server.

The Web UI makes a call to the Codec Server only when it needs to encode/decode data  before/after API calls to the Cluster.
@robholland ^ is this accurate?

Reply:
Thank you, Rob! This really clears things up

Reply:
@robholland Should I correct the diagrams?